src/open_inwoner/ssd/xml.py

```diff
@@ -43,17 +47,17 @@ def format_date_month_name(date_str) -> str:
 
 
 def format_name(first_name: str, voorvoegsel: str, last_name: str):
-    first_names = first_name.split(" ")
-    first_name_initials = [name[0] + "." for name in first_names]
+    first_names = first_name.strip().split(" ")
+    first_name_initials = (name[0] + "." for name in first_names)
     first_name_formatted = " ".join(first_name_initials)
 
-    if voorvoegsel:
-        return f"{first_name_formatted} {voorvoegsel} {last_name}"
-    return f"{first_name_formatted} {last_name}"
+    elements = (first_name_formatted, voorvoegsel, last_name)
+
+    return " ".join((item.strip() for item in elements if item))
```

Comment:
This is not correct. The conditions is applied before the left side of the loop:

```py
ll = ['a', 'b', '', '   ', 'c']
[a.strip() for a in ll if a]
-> ['a', 'b', '', 'c']
```

So strings that .strip() to an empty string are not seen and filtered by the condition.